v1alpha1/proto/grafeas.proto

```diff
@@ -762,6 +817,35 @@ message Occurrence {
 
   // Output only. The time this `Occurrence` was last updated.
   google.protobuf.Timestamp update_time = 10;
+
+  // Describes how this occurrence can be verified
+  oneof verification {
+    IntotoVerification intoto = 16;
+  }
+}
+
+// Includes information about how to verify an occurrence using in-toto
+// TODO(r2wend2): This copies some information from the occurrence itself need to reconcile before merging.
```

Comment:
Maybe I'm reading this comment wrong, but I wouldn't say that in-toto links (AFAICS described by this message) include information about how to verify in-toto occurrences (layouts). They actually provide the required signed evidence (which functionary, used command, materials, products, environment, ...) in order to verify that the supply chain was carried as specified in the corresponding in-toto layout.